pyproject.toml

```diff
@@ -1,7 +1,92 @@
 [build-system]
-requires = ["setuptools", "wheel"]
```

Comment:
Are you sure we don't need to require wheel?

Reply:
Yes. Looking for documentation, but setuptools after (I want to say) version 61.0.0 includes wheel.

Reply:
As a side note, but not something I ever do with projects because of readability, if using `setuptools` as used here, one doesn’t technically need the `build-system` section since `pyproject` uses it by default. Of course, it’s much easier to just include it and more explicit to know what version of `setuptools` you’re using.

Reply:
> The setuptools package implements the build_sdist command and the wheel package implements the build_wheel command; **the latter is a dependency of the former exposed via [PEP 517](https://peps.python.org/pep-0517/)** hooks.

Ref: https://setuptools.pypa.io/en/latest/build_meta.html

Reply:
For a follow-up PR... Add [pyproject-fmt](https://pypi.org/project/pyproject-fmt/) and [validate-pyproject](https://pypi.org/project/validate-pyproject/) to [.pre-commit-config.yaml](https://github.com/pypa/pipenv/blob/main/.pre-commit-config.yaml).

Reply:
@cclauss Please add this to issue #5906 since this also deals with a follow-up to this issue.